openedx2zim/instance_connection.py

```diff
@@ -34,6 +22,36 @@ def __init__(self, email, password, instance_config, locale):
         self.instance_connection = None
         self.user = None
         self.locale = locale
+        self.build_dir = build_dir
+        self.debug = debug
+
+    def get_response(self, url, post_data, headers, max_attempts=5):
+        req = urllib.request.Request(url, post_data, headers)
```

Comment:
I see previous code was using urllib. Unless there's a reason to use it, we should introduce `requests` and use it.

Reply:
You can open a ticket for it though. As code itself didn't change, we don't need to switch **now**

Reply:
Tracked in #180